vanhouse/src/App.js

```diff
@@ -27,6 +28,7 @@ function App() {
                   <UserList
                     setReset={setReset}
                     setQuery={setFilterURL}
+                    setUserId={setUserId}
                   />
                 )) ||
                   (filterIdx === 1 && (
```

Comment:
@tyxxxx Do we still need to keep `<Location />`, `<Price />` and `<Category />`?